cloudhub-2/modules/ROOT/pages/ch2-manage-schedules.adoc

```diff
@@ -25,27 +25,36 @@ To view the application schedules, you must have the *Exchange Viewer* permissio
 
 When designing the schedule, consider the following:
 
+
+* Plan for high availability during network failures. CloudHub 2.0 infrastructure restarts due to hardware issues or scheduled maintenance, and security updates.
++
+During service updates or security patching, xref:mule-runtime::global-settings-configuration.adoc#global-configurations-reference[CloudHub 2.0 waits five minutes] for existing schedules to complete. After a new replica is launched, the schedule runs at its next due. 
```

Comment:
```suggestion
During service updates or security patching, xref:mule-runtime::global-settings-configuration.adoc#global-configurations-reference[CloudHub 2.0 waits five minutes] for existing schedules to complete. After a new replica is launched, the schedule runs at its next scheduled time. 
```

Reply:
Why the inline link? I think in this case, it would be better to have a "For more information about <whatever this link is meant to convey>, see xref:mule-runtime::global-settings-configuration.adoc#global-configurations-reference[Global Configurations Reference].

Reply:
this should be the same behavior as described in cloudhub-2/modules/ROOT/pages/ch2-understand-app-restart.adoc. - Cloudhub 2.0 waits at max five minutes only if shutdownTimeout is configured properly